draft-ietf-acme-acme.md

```diff
@@ -191,7 +192,7 @@ from the client.
 
 ACME allows a client to request certificate management actions using a set of
 JavaScript Object Notation (JSON) messages carried over HTTPS.  In many ways,
-ACME functions much like a traditional CA, in which a user creates an account,
+issuance using ACME proceeds in much way as a traditional issuance process, in which a user creates an account,
```

Comment:
typo: "much way as a" -> "much in the same way as a" ?

I think using "issuance process" again at the tail end of this first clause is confusing. Maybe instead:
"issuance using ACME proceeds in much the same way as a traditional CA's issuance process" ?

Writing this out make me think this sentence is a) long b) overly complex for what its saying.

Another idea:
"ACME resembles a traditional CA's issuance process in which a user creates an account, ..."